src/domain/modules/modules.repository.interface.ts

```diff
@@ -0,0 +1,15 @@
+import { SafesByModule } from '@/domain/modules/entities/safes-by-module.entity';
+
+export const IModulesRepository = Symbol('IModulsRepository');
+
+export interface IModulesRepository {
+  getSafesByModule(args: {
+    chainId: string;
+    moduleAddress: string;
+  }): Promise<SafesByModule>;
+
+  clearSafesByModule(args: {
+    chainId: string;
+    moduleAddress: string;
+  }): Promise<void>;
```

Comment:
Nitpick: no strong opinion regarding this, but given these methods retrieve/clear Safe addresses and not modules, do you think it may have sense to have them on `SafeRepository` instead of creating this new `ModulesRepository`?

Reply:
I've migrated it in cbc68a8b4ec8699a56ccdd1a95c5226325bec844.